sonata-project/page-bundle/3.9/config/packages/sonata_page.yaml

```diff
@@ -0,0 +1,87 @@
+sonata_block:
+    context_manager: sonata.page.block.context_manager
+    default_contexts: [sonata_page_bundle]
+
+sonata_admin:
```

Comment:
Do we really need to modify the config for other bundles?

I prefer to review 2 more PRs with recipes for these bundles.

Reply:
SonataPageBundle will break SonataAdminBundle without this config. These bundles work together; also PageBundle cannot work without AdminBundle

Reply:
Okey, makes sense.